src/middlewared/middlewared/plugins/acme_protocol_/authenticators/factory.py

```diff
@@ -5,6 +5,7 @@
 from .cloudflare import CloudFlareAuthenticator
 from .route53 import Route53Authenticator
 from .shell import ShellAuthenticator
+from .ovh import OVHAuthenticator
```

Comment:
While we are making some other changes, can we please have this alphabetically ordered :D 

So it would come after `cloudflare`, similar changes would be required below